app/src/main/java/com/infomaniak/swisstransfer/ui/images/illus/onboarding/StorageBoxPileLight.kt

```diff
@@ -0,0 +1,633 @@
+package com.infomaniak.swisstransfer.ui.images.illus.onboarding
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.EvenOdd
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.group
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.infomaniak.swisstransfer.ui.images.AppImages
+import com.infomaniak.swisstransfer.ui.images.AppImages.AppIllus
+
+val AppIllus.StorageBoxPileLight: ImageVector
+    get() {
+        if (_storageBoxPileLight != null) {
+            return _storageBoxPileLight!!
+        }
+        _storageBoxPileLight = Builder(
+            name = "StorageBoxPileLight",
+            defaultWidth = 290.0.dp,
+            defaultHeight = 258.0.dp,
+            viewportWidth = 290.0f,
+            viewportHeight = 258.0f
+        ).apply {
+            path(
+                fill = SolidColor(Color(0xFFA98116)),
+                stroke = null,
+                strokeLineWidth = 0.0f,
+                strokeLineCap = Butt,
+                strokeLineJoin = Miter,
+                strokeLineMiter = 4.0f,
+                pathFillType = EvenOdd
+            ) {
+                moveTo(98.03f, 94.1f)
+                curveToRelative(0.78f, 0.68f, 3.34f, 1.72f, 4.57f, 1.98f)
+                curveToRelative(-1.3f, -0.57f, -3.01f, -1.2f, -3.95f, -1.99f)
+                curveToRelative(-4.49f, -3.77f, -1.84f, -10.05f, 3.96f, -9.13f)
+                curveToRelative(1.06f, 0.17f, 2.86f, 1.02f, 4.31f, 1.73f)
+                curveToRelative(-0.97f, -0.69f, -3.82f, -2.05f, -4.84f, -2.28f)
+                curveToRelative(-4.83f, -1.07f, -9.71f, 4.74f, -4.05f, 9.68f)
+            }
+            group {
```

Comment:
This group looks like it's useless, and it's not the only one in this file.

Reply:
It's probably the same in the following illustrations, I'll let you check them.

Reply:
About groups in general, I have answered to the same comment here: https://github.com/Infomaniak/android-SwissTransfer/pull/222#discussion_r1870905083